src/style.css

```
@@ -0,0 +1,53 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  margin: 0 8px;
+}
+
+.search-bar {
+  font-family: Avenir, Arial, sans-serif;
+  width: 100%;
+  margin-top: 20px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  color: #3d4e61;
+
+  background: url(images/Search.svg);
+  background-repeat: no-repeat;
+}
+
+.big-search-bar {
+  font-size: 16px;
+  height: 70px;
+  padding-left: 62px;
+  background-position: 26px 50%;
+  background-size: 19px 19px;
+}
+
+.small-search-bar {
+  font-size: 14px;
+  height: 42px;
+  padding-left: 33px;
+  background-position: 13px 50%;
+  background-size: 11px 11px;
+}
+
+.search-bar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```

```suggestion
  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%) url("images/Search.svg");
```
Спробуй для фокуса також зображення вказувати. Тоді не буде зникати